nsq/async.py

```diff
@@ -520,3 +530,20 @@ def _on_message_touch(self, message, **kwargs):
                 conn=self,
                 error=protocol.SendError('failed to send TOUCH %s' % message.id, e),
             )
+
+    def __clear_timeout(self, message):
```

Comment:
I think all of these related timeout functions and variables need to be more specifically named, i.e. `s/timeout/message_timeout`